test/physical/test_common.c

```diff
@@ -114,14 +119,22 @@ void testsuite_stopBidib(void) {
 }
 
 void testsuite_signal_callback_handler(int signum) {
+	printf("testsuite: SIG %d - before stopping, debug logs:\n", signum);
+	printf("   Track output states:\n");
+	testsuite_logAllTrackOutputStates();
+	printf("\n");
+	printf("   Booster power states:\n");
+	testsuite_logAllBoosterPowerStates();
+	printf("\n");
+	printf("testsuite: SIG %d - now stopping libbidib.\n", signum);
 	testsuite_stopBidib();
-	printf("testsuite: SIGINT - stopping libbidib \n");
+	printf("testsuite: SIG %d - libbidib stopped.\n", signum);
```

Comment:
Instead of printing the signal code, you could use `strsignal` or `sys_signame` to print the signal name: https://stackoverflow.com/a/16509837